src/components/Reports/ReportsDailyVisits/ReportsDailyVisits.js

```diff
@@ -18,23 +18,54 @@ const ReportsDailyVisits = ({ domainName, dateFrom, dependencies: { datahubClien
       show: false,
     },
     tooltip: {
-      format: {
-        title: (date) => {
-          return intl.formatDate(date, {
+      contents: function(data) {
+        if (data.length) {
+          const date = intl.formatDate(data[0].x, {
             timeZone: 'UTC',
             day: 'numeric',
             month: 'long',
             year: 'numeric',
             weekday: 'long',
           });
-        },
-        value: (value) => {
-          return (
-            intl.formatMessage({
-              id: 'reports_daily_visits.tooltip_page_views',
-            }) + value
-          );
-        },
+
+          const tooltipData = data.reduce((accumulator, item) => {
+            accumulator[item.id] = item.value;
+            return accumulator;
+          }, {});
+
+          return `
+          <div class="c3-tooltip">
+            <div class='tooltip-title'>${date}</div>
+            <div class='tooltip-value'>
+                ${
+                  !tooltipData.withEmail
+                    ? intl.formatMessage({
+                        id: 'reports_daily_visits.tooltip_page_views',
+                      }) + tooltipData.quantity
+                    : ''
+                }
+            </div>
```

Comment:
Is it fine having this element empty?

```html
            <div class='tooltip-value'>
            </div>
```

Maybe the _div_ should not exist. What do you think about nesting HTML inside template entries or pre-define them before?